demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -158,8 +157,7 @@ def _get_resource_path_from_source_control(
             KeyError,
             IndexError,
         ) as e:
-            msg = f"{e.__class__.__name__}: Could not find file '{str(resource_path)}' in {'remote' if remote else 'local'}. Please specify the full path to the integration YAML file, e.g. `demisto-sdk generate-docs -i $(realpath {resource_path})`"
-            logger.error(msg)
+            msg = f"{e.__class__.__name__}: Could not find file '{str(resource_path)}' in {'remote' if remote else 'local'}."
```

Comment:
```suggestion
            msg = f"{e.__class__.__name__}: Could not find file '{resource_path}' in {'remote' if remote else 'local'}."
```

When using a variable inside f-string it automatically calls the `__str__` method of an object if exist.